diesel/src/connection/mod.rs

```diff
@@ -40,7 +40,17 @@ pub trait SimpleConnection {
 ///
 /// Users should threat this type as `impl Iterator<Item = QueryResult<impl Row<DB>>>`
 pub type LoadRowIter<'conn, 'query, C, DB, B = DefaultLoadingMode> =
```

Comment:
This type can be deprecated. (It should be behind the `with-deprecated` feature flag + removed for the `without-deprecated` flag)